ui/app/templates/vault/error.hbs

```diff
@@ -1,27 +1,45 @@
-<section class="section">
-  <div class="container">
-    {{#if (eq this.model.httpStatus 404)}}
-      <NotFound @model={{this.model}} />
-    {{else}}
-      <PageHeader as |p|>
-        <p.levelLeft>
-          <h1 class="title is-3 has-text-grey">
-            {{#if (eq this.model.httpStatus 403)}}
-              Not authorized
-            {{else}}
-              Error
-            {{/if}}
-          </h1>
-        </p.levelLeft>
-      </PageHeader>
-      <BlockError>
-        {{#if this.model.message}}
-          <p>{{this.model.message}}</p>
-        {{/if}}
-        {{#each this.model.errors as |error|}}
-          <p>{{error}}</p>
-        {{/each}}
-      </BlockError>
-    {{/if}}
+<div class="is-flex-1 is-flex-v-centered">
+  <div class="empty-state-content">
+    <div class="is-flex-v-centered has-bottom-margin-xxl">
+      <div class="brand-icon-large">
+        <Icon @name="vault" @size="24" @stretched={{true}} />
+      </div>
+    </div>
+    <div class="is-flex-center">
+      <div class="error-icon">
+        <Icon @name="help" @size="24" class="has-text-grey" @stretched={{true}} />
+      </div>
+      <div class="has-left-margin-s">
+        <h1 class="is-size-4 has-text-semibold has-text-grey has-line-height-1">
+          {{#if (eq this.model.httpStatus 403)}}
+            Not authorized
+          {{else if (eq this.model.httpStatus 404)}}
+            Page not found
+          {{else}}
+            Error
+          {{/if}}
+        </h1>
+        <p class="has-text-grey is-size-8">Error {{this.model.httpStatus}}</p>
+      </div>
+    </div>
+
+    <p class="has-text-grey has-top-margin-m has-bottom-padding-l has-border-bottom-light">
+      {{#if (eq this.model.httpStatus 404)}}
+        Sorry, we were unable to find any content at that URL. Double check it or go back home.
+      {{else}}
+        {{this.model.message}}
+        {{join ". " this.model.errors}}
```

Comment:
I know sometimes we see the `[object Object]` error message (on the old one) -- this may be out of scope, but I wonder if a helper to process `this.model.errors` into something readable would help?

Reply:
I think that's a good suggestion and we should have a common way to handle API errors. We do have the `error-message` utility that could be turned into a helper. It might not be relevant here since I'm not sure this block will actually be hit. I kept it since it was a part of the original implementation. In the `cluster` route there is sort of a pseudo API error being returned:
```js
return reject({ httpStatus: 404, message: 'not found', path: params.cluster_name });
```
Do you think I should get rid of the other status handling and simplify the template or leave it just in case?

Reply:
I'm going to merge this but happy to make further changes as part of the primary PR.